.github/workflows/build.yml

```diff
@@ -1,109 +1,34 @@
-
-      - name: Publish to Maven Central
-        id: publish-to-maven-central
-        if: ${{ github.event_name != 'pull_request' && github.ref == 'refs/heads/main'}}
-        run: ./mvnw clean deploy -Drelease=true
-        env:
-          OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
-
-  Perform-Release:
-    runs-on: ubuntu-latest
-    needs: Build-and-Publish
-    if: ${{ !contains(needs.Build-and-Publish.outputs.version, 'SNAPSHOT') && github.event_name != 'pull_request' && github.ref == 'refs/heads/main'}}
-
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          submodules: 'recursive'
-
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v2
-        with:
-          distribution: adopt-hotspot
-          java-version: 8
-          cache: maven
-
-      - name: Tag Release
-        env:
-          GIT_USER: ${{ secrets.GIT_USER }}
-          GIT_PASSWORD: ${{ secrets.GIT_TOKEN }}
-        run: |
-          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
-          git config --global user.name "${{ github.actor }}"
-          git remote set-url origin "https://${{github.actor}}:${{ github.token }}@github.com/${{ github.repository }}.git"
-          git tag -a "v${{ steps.set-version.outputs.version }}" "${{ github.sha }}"  -m "Version ${{ steps.set-version.outputs.version }}"
-          git push --follow-tags
-
-      - name: Increment Version
-        run: |
-          ./mvnw clean build-helper:parse-version versions:set -DnewVersion=\${parsedVersion.majorVersion}.\${parsedVersion.minorVersion}.\${parsedVersion.nextIncrementalVersion}-SNAPSHOT -DprocessAllModules versions:commit
-
-      - name: Set New Version Variable
-        id: set-new-version
-        run: echo ::set-output name=version::$(./mvnw -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v4
-        with:
-          branch: "feature/v${{ steps.set-new-version.outputs.version }}"
-          commit-message: "[create-pull-request] Auto increment to v${{ steps.set-new-version.outputs.version }}"
-          title: "Auto increment to v${{ steps.set-new-version.outputs.version }}"
-          delete-branch: true
-          assignees: ${{ github.actor }}
-          reviewers: adesjardin, manikmagar, kkingavio
\ No newline at end of file
+  Build-Maven:
+    uses: avioconsulting/shared-workflows/.github/workflows/maven-build.yml@main
+    secrets: inherit
+    with:
+      include-mule-ee-repo: true
+
+  Release-Maven:
+    needs: Build-Maven
+    uses: avioconsulting/shared-workflows/.github/workflows/maven-release.yml@main
+    secrets: inherit
+    with:
+      app-version: ${{ needs.Build-Maven.outputs.app-version }}
+      publish-maven-central: false
```

Comment:
@adesjardin I have turned this off since jreleaser.yml is configured to publish to maven central.

Reply:
Yes, we only added that because jreleaser couldn't publish mule plugins correctly to maven central.